internal/v1/api.go

```diff
@@ -182,11 +182,15 @@ const (
 	ImageTypes_aws                 ImageTypes = "aws"
 	ImageTypes_azure               ImageTypes = "azure"
 	ImageTypes_edge_commit         ImageTypes = "edge-commit"
+	ImageTypes_edge_container      ImageTypes = "edge-container"
```

Comment:
so this was disabled anyway the whole time :|

Reply:
looks like it.  which reminds me, it also needs to be added to the upload target switch.